docs/source/quick_start/install.rst

```diff
@@ -0,0 +1,254 @@
+
+
+Using virtual environment
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For the most native, but more complicated option, you can create a virtual environment and install MCDR in it
+
+Create a virtual environment by:
+
+.. prompt:: bash
+
+    python3 -m venv <venv directory>
+
+``venv`` for example:
+
+.. prompt:: bash
+
+    python3 -m venv venv
+
+Activate it by:
+
++----------+------------+-----------------------------------------+
+| Platform | Shell      | Command to activate virtual environment |
++==========+============+=========================================+
+|  POSIX   | bash/zsh   | $ source venv/bin/activate              |
++          +------------+-----------------------------------------+
+|          | fish       | $ source venv/bin/activate.fish         |
++          +------------+-----------------------------------------+
+|          | csh/tcsh   | $ source venv/bin/activate.csh          |
++          +------------+-----------------------------------------+
+|          | PowerShell | $ venv/bin/Activate.ps1                 |
++----------+------------+-----------------------------------------+
+| Windows  | cmd.exe    | > venv\\Scripts\\activate.bat           |
++          +------------+-----------------------------------------+
+|          | PowerShell | PS > venv\\Scripts\\Activate.ps1        |
++----------+------------+-----------------------------------------+
+
+Then, install MCDR using pip:
+
+.. prompt:: bash
+    :prompts: (venv) $
+
+    pip install mcdreforged
+
+When a new version of MCDR available, you may upgrade MCDR by:
+
+.. prompt:: bash
+    :prompts: (venv) $
+
+    pip install mcdreforged -U
+
+An animated demo with bash:
+
+.. asciinema:: resources/venv.cast
+    :rows: 10
+
+|
+
+.. note::
+
+    In this way, you must activate the virtual environment every time you want to use MCDR, or install packages for MCDR plugins
+
+.. seealso ::
+
+    Python Doc: `How venvs work <https://docs.python.org/3/library/venv.html#how-venvs-work>`__
+
+Using Docker
+~~~~~~~~~~~~
+
+MCDR also provides Docker images as an option. See :doc:`/docker` for more details
+
+Specify MCDR version by the tag of Docker image
+
+Compared to the two methods above, Docker has a more convoluted learning path, but convenient for some advanced usages
+
+System package manager
+~~~~~~~~~~~~~~~~~~~~~~
+
+You may found MCDR in some package repositories, AUR for example. However, it's **definately not recommended** to use a system package manager to install MCDR. Not only it has all the same problem as the system-wide pip install, it's also hard to manage the dependencies of MCDR plugins
+
+Accelerate the installation
+---------------------------
+
+For users in China, you may use a mirror to accelerate ``pip`` and ``pipx``. Here's an example with 
+`Tsinghua University TUNA mirror <https://mirrors.tuna.tsinghua.edu.cn/help/pypi/>`__:
+
```

Comment:
imo, just point out that you need to add a `-i https://pypi.tuna.tsinghua.edu.cn/simple` to the package installation command. User might not get what your example want to express

Reply:
```rst
Let's use `Tsinghua University TUNA mirror <https://mirrors.tuna.tsinghua.edu.cn/help/pypi/>`__ for example
```
sounds better

Reply:
The changes still does not express what's the key difference when using the pip mirror  -- the `-i xxxx` argument